src/js/components/stock-movement/modals/RejectRequestModalButtons.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+import Translate from 'utils/Translate';
+
+const RejectRequestModalButtons = ({ closeRejectionModal }) => (
+  <div className="btn-toolbar justify-content-between pt-3">
+    <button
+      type="button"
+      className="btn btn-outline-primary ml-1"
+      onClick={closeRejectionModal}
+    >
+      <Translate id="react.rejectRequestModal.cancel.label" defaultMessage="Cancel" />
+    </button>
+    <button
+      type="submit"
+      className="btn btn-primary align-self-end"
+    >
+      <Translate id="react.rejectRequestModal.confirmReject.label" defaultMessage="Confirm reject" />
+    </button>
```

Comment:
I don't know how those buttons look like (it'd be good if you attached a screenshot of the modal), but I'm wondering if it would be possible maybe to use our reusable `Button` component for that using a proper `variant` prop?

Reply:
That's the same button, so I am changing it.
![image](https://github.com/openboxes/openboxes/assets/83239466/bdb7142d-35d0-432d-ae0e-c9f806330b28)